docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -338,11 +349,41 @@ and *W* is limited to 87 when there are 64 input channels.
 
 .. note:: 
 
-   The above limitations only apply to the new hardware 2D convolution operation. When possible, the Intel® GNA 
+   The above limitations only apply to the new hardware 2D convolution operation. For GNA 3.0 When possible, the Intel® GNA
```

Comment:
Is "the new operation" a correct term here, when we're describing older HW generation?

Also:
```suggestion
   The above limitations only apply to the new hardware 2D convolution operation. For GNA 3.0, when possible, the Intel® GNA
```

Reply:
Convolution is new to GNA 3.x.